linkml_model/model/schema/aliases.yaml

```diff
@@ -0,0 +1,84 @@
+id: https://w3id.org/linkml/mappings
+title: LinkML Schema Mappings
+name: mappings
+description: LinkML model for mappings
+license: https://creativecommons.org/publicdomain/zero/1.0/
+version: 2.0.0
+
+prefixes:
+  linkml: https://w3id.org/linkml/
+  skos: http://www.w3.org/2004/02/skos/core#
+  OIO: http://www.geneontology.org/formats/oboInOwl#
+  IAO: http://purl.obolibrary.org/obo/IAO_
+
+default_curi_maps:
+    - semweb_context
+
+default_prefix: linkml
+default_range: string
+
+emit_prefixes:
+  - linkml
+  - rdf
+  - rdfs
+  - xsd
+  - skos
+  - OIO
+  - IAO
+
+imports:
+  - linkml:types
+  - linkml:meta
+
+slots:
+  alias_source:
+    range: string
+    required: true
+    description: >-
+      Can be a URIOrCurie that represents the source of the alias, or it can be a simple string that
+      describes the source of the alias.
+
+  alias_scope:
+    range: alias_scope_enum
+
+  alias:
+    domain: slot_definition
+    range: string
+    slot_uri: skos:prefLabel
+    description: >-
+      the name used for a slot in the context of its owning class.  If present, this is used instead of the actual
+      slot name.
+
+  aliases:
+    range: alias_definition
```

Comment:
unfortunately this would be a breaking change, we will need to have a different slot name here, like structured_aliases

yes, this means there are potentially duplicative ways of assigning aliases, but I think we have to live with that until we move to linkml2.0